From 6560248ee6e7de1e1bcc8c19c57ca60cc87a4157 Mon Sep 17 00:00:00 2001
From: John Trupiano <jtrupiano@gmail.com>
Date: Sun, 27 Sep 2009 10:15:17 -0400
Subject: [PATCH] Allow PKG_NAME env variable to control the directory name used
 when 'rake dev' is used to generate a new rails app.

---
 railties/Rakefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/railties/Rakefile b/railties/Rakefile
index b7c199f..c11c84f 100644
--- a/railties/Rakefile
+++ b/railties/Rakefile
@@ -9,7 +9,7 @@ require 'rbconfig'
 require File.join(File.dirname(__FILE__), 'lib/rails', 'version')
 
 PKG_BUILD       = ENV['PKG_BUILD'] ? '.' + ENV['PKG_BUILD'] : ''
-PKG_NAME        = 'rails'
+PKG_NAME        = ENV['PKG_NAME'] || 'rails'
 PKG_VERSION     = Rails::VERSION::STRING + PKG_BUILD
 PKG_FILE_NAME   = "#{PKG_NAME}-#{PKG_VERSION}"
 PKG_DESTINATION = ENV["RAILS_PKG_DESTINATION"] || "../#{PKG_NAME}"
-- 
1.6.1

